build/build.ps1

```diff
@@ -115,6 +115,9 @@ function Build {
   $useOpenCover = $useCodecov
 
   & $MsbuildExe $ToolsetBuildProj /m /nologo /clp:Summary /nodeReuse:$nodeReuse /warnaserror /v:$verbosity $logCmd /p:Configuration=$configuration /p:SolutionPath=$solution /p:Restore=$restore /p:QuietRestore=true /p:DeployDeps=$deployDeps /p:Build=$build /p:Rebuild=$rebuild /p:Deploy=$deploy /p:Test=$test /p:IntegrationTest=$integrationTest /p:Sign=$sign /p:Pack=$pack /p:UseCodecov=$useCodecov /p:UseOpenCover=$useOpenCover /p:CIBuild=$ci /p:NuGetPackageRoot=$NuGetPackageRoot $properties
```

Comment:
There is another `$MsbuildExe` invocation below that seems like it should be similarly protected.

Reply:
It's protected via the call site, which is how the original call was protected before I added code coverage support.

Reply:
The protection for the `$MsbuildExe` invocation on line 124 seems very fragile. True it's the last line in the `Build` function and we immediately exit after `Build` using `$lastExitCode`. But that seems very fragile to change.